test/test_metadata.py

```diff
@@ -87,13 +87,6 @@ def test_metadata_implicit_delete(self):
         self.assertNotIn("unknown", self.metadata)
         self.assertNotIn("unknown", self.metadata.deleted_tags)
 
-    def test_metadata_set_explicit_empty(self):
```

Comment:
I could not find any usage of this in real code, so test gets dropped. Interestingly all other tests continue to work without issue, which kind of shows how the "values are strings" assumption is really inherit to the code.